lib/macho/macho_file.rb

```diff
@@ -424,6 +463,16 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
+      fmt = Utils.specialize_format("L=", endianness)
+      new_ncmds = [ncmds].pack(fmt)
```

Comment:
Yeah, that's what I meant. Sorry for the confusion caused by referencing `ncmds` instead of `new_ncmds` in my very first comment here. Glad we sorted this out. 😄